test/integration/gardenlet/seed/seed/seed_test.go

```diff
@@ -512,8 +513,8 @@ var _ = Describe("Seed controller tests", func() {
 							"dnsrecords.extensions.gardener.cloud",
 							"extensions.extensions.gardener.cloud",
 							// etcd-druid
-							"etcds.druid.gardener.cloud",
-							"etcdcopybackupstasks.druid.gardener.cloud",
+							druidcorecrds.ResourceNameEtcd,
+							druidcorecrds.ResourceNameEtcdCopyBackupsTask,
```

Comment:
Revert this change - it is a test file.

Reply:
addressed in https://github.com/gardener/gardener/pull/11545/commits/cdf2536229ecc8936f22fd06d7c857081cb7932b